commander/package.json

```diff
@@ -121,45 +117,43 @@
 		"listr": "0.14.3",
 		"lockfile": "1.0.4",
 		"pm2": "3.3.1",
-		"semver": "5.6.0",
+		"semver": "5.5.1",
```

Comment:
why downgrading dependencies from what is there in development? (for most of the changes in this package.json)

Reply:
It's just to have consistency across the packages